packages/iov-tendermint-rpc/types/types.d.ts

```diff
@@ -0,0 +1,6 @@
+import { As } from "type-tagger";
```

Comment:
I don't really see why you pulled this out from encoding, as they are specific encoding types, but not a blocker for me.

Reply:
I think this belongs to responses, but I did not want to write `response.TxHash` all the time, especially because there are different response implementations.